src/queries/article/index.ts

```diff
@@ -119,6 +121,8 @@ export default {
     revisionCount,
     license: ({ license }: { license?: GQLArticleLicenseType }) =>
       license || ARTICLE_LICENSE_TYPE.cc_by_nc_nd_2,
+    supportRequest,
```

Comment:
Can omit resolvers of `supportRequest` and `supportReply` if the return value is inherited from database value directly.

Reply:
For `supportReply` ,  some validation code will be added to check viewer if a donator in another PR.   And I hesitate to  add extra code to  handle native article problem here.